### DISCUSSION TITLE
Data transformation functions such as angular pipes feature support

### DISCUSSION THREAD

**Discussion by savanrajkotiya** [solidjs/solid#1625] [Q&A]
Are there any equivalent features to Angular pipes in the solid js?
However, I can create custom functions or components that perform similar operations to Angular pipes, such as formatting dates or numbers, filtering lists, or transforming strings. These functions or components can be reused across multiple components as needed but is it optimised & better way ?
 Or Is there any hidden method or feature available in solid js with use of it I can do this things in very optimised and effective way.
 
 ex. currently doing something like this in solid js.
```
 
 // custom fn that takes date value & return desired formatted date
  const  formatDate = (value: string) => 
  { return /* ... */ } 
  
  
  /* in JSX template */
  <span> formatDate(dateValue) </span>
```

**Comment by ryansolid** [comment]
I'm not sure what the question is as I'm not super familiar with Angular Pipes. You can create functions that you could re-use if you want to.

Is it just a pipe operator syntax? If so we won't have support for this until it is added to the JS language as that is the limitation we have within expressions with JSX.

Are there any other benefits to it?
